types/big.js/test/big.js-module-tests.ts

```diff
@@ -6,7 +6,7 @@
   Minor changes have been made such as adding variable definitions where required.
 */
 
-import { Big, RoundingMode } from 'big.js';
+import { Big, RoundingMode } from "big.js";
```

Comment:
```suggestion
import { Big, RoundingMode } from 'big.js';
```
The file uses single ones